src/circuit.rs

```diff
@@ -8,14 +8,18 @@
 
 use core::marker::PhantomData;
 
-use crate::commitment_scheme::kzg10::PublicParameters;
 use crate::constraint_system::StandardComposer;
 use crate::error::Error;
-use crate::proof_system::{Proof, Prover, ProverKey, Verifier, VerifierKey};
-use ark_ec::models::twisted_edwards_extended::{GroupAffine, GroupProjective};
+use crate::proof_system::{
+    Proof, Prover, ProverKey, Verifier, VerifierKey as PlonkVerifierKey,
```

Comment:
Is there a need to separate verifier key and plonk verifier key?

Reply:
Right now we are overusing the name 'VerifierKey'. It refers to the commitments of arith polynomials, bool polynomials, etc. as well as the union of all of these keys used to verify the whole circuit.
With this change to the poly commit we have another VerifierKey so I decided to use an alias for the other one.
This problem is addressed in Issue #14